### PR TITLE
Fix: Update 'Need Help' link for grid container [ED-12592]

### PR DESCRIPTION
**Pull request by ronenboxer**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by louiswol94 — CHANGES_REQUESTED** [review]
Hey @ronenelementor , this was a good deep dive ;)
But the controls stack is a base that should not be aware of any specific feature (like the container).
Also your solution does not work when the "Container Layout" option is changed.
I suggest you look into editing `/assets/dev/js/editor/elements/views/container.js`

**Review by louiswol94 — DISMISSED** [review]

File: tests/playwright/sanity/container-grid.test.ts [hunk 1]
```diff
@@ -651,6 +651,57 @@ test.describe( 'Container Grid tests @container', () => {
 			await expect( editor.getPreviewFrame().locator( '.elementor-empty-view' ) ).toBeVisible();
 		} );
 	} );
+
+	test( 'Need Help url for grid and flex containers', async ( { page }, testInfo ) => {
+		// Arrange
+		const wpAdmin = new WpAdminPage( page, testInfo );
```
Comment (hein-obox) [inline tests/playwright/sanity/container-grid.test.ts:657]:
```suggestion
		const wpAdmin = new WpAdminPage( page, testInfo ),
```

File: tests/playwright/sanity/container-grid.test.ts [hunk 2]
```diff
@@ -651,6 +651,57 @@ test.describe( 'Container Grid tests @container', () => {
...
+		const editor = await wpAdmin.openNewPage(),
```
Comment (hein-obox) [inline tests/playwright/sanity/container-grid.test.ts:658]:
```suggestion
		editor = await wpAdmin.openNewPage(),
```

File: tests/playwright/sanity/container-grid.test.ts [hunk 3]
```diff
@@ -651,6 +651,57 @@ test.describe( 'Container Grid tests @container', () => {
...
+			let linkElement = page.locator( '#elementor-panel__editor__help__link' );
+			expect.soft( linkElement ).toHaveAttribute( 'href', 'https://go.elementor.com/widget-container' );
+		} );
+
+		await test.step( 'Changing layout to grid', async () => {
+			// Act
+			await editor.setSelectControlValue( 'container_type', 'grid' );
+
+			// Assert
+			let linkElement = page.locator( '#elementor-panel__editor__help__link' );
```
Comment (hein-obox) [inline tests/playwright/sanity/container-grid.test.ts:677]:
```suggestion
			const linkElement = page.locator( '#elementor-panel__editor__help__link' );
```

File: tests/playwright/sanity/container-grid.test.ts [hunk 4]
```diff
@@ -651,6 +651,57 @@ test.describe( 'Container Grid tests @container', () => {
 			await expect( editor.getPreviewFrame().locator( '.elementor-empty-view' ) ).toBeVisible();
 		} );
 	} );
+
+	test( 'Need Help url for grid and flex containers', async ( { page }, testInfo ) => {
+		// Arrange
+		const wpAdmin = new WpAdminPage( page, testInfo ),
+		editor = await wpAdmin.openNewPage(),
```
Comment (hein-obox) [inline tests/playwright/sanity/container-grid.test.ts:658]: Indentation.

File: tests/playwright/sanity/container-grid.test.ts [hunk 5]
```diff
@@ -651,6 +651,57 @@ test.describe( 'Container Grid tests @container', () => {
...
+			frame = editor.getPreviewFrame(),
+			containerId = await editor.addElement( { elType: 'container' }, 'document' );
+		await editor.selectElement( containerId );
+
+		await test.step( 'Changing layout to flex', async () => {
+			// Act
+			await editor.setSelectControlValue( 'container_type', 'flex' );
+
+			// Assert
+			let linkElement = page.locator( '#elementor-panel__editor__help__link' );
```
Comment (hein-obox) [inline tests/playwright/sanity/container-grid.test.ts:668]:
```suggestion
			const linkElement = page.locator( '#elementor-panel__editor__help__link' );
```

Comment (ronenelementor) [reply tests/playwright/sanity/container-grid.test.ts]: Done

File: tests/playwright/sanity/container-grid.test.ts (hunk 4 above)
Comment (ronenelementor) [reply tests/playwright/sanity/container-grid.test.ts]: Done

**Review by hein-obox — APPROVED** [review]

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**